src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -400,6 +409,8 @@ public void Dispose()
         }
 
         private static readonly string CachedDatabaseInfo = "CachedDatabaseInfo";
+        public EventHandler<NonDurabilitySupportEventArgs> NonDurableFileSystemError;
```

Comment:
Huh? Why are you putting the event here? It should be on the actual `StorageEnvironmentOptions`, no?